blocks/carousel/carousel.js

```diff
@@ -1,51 +1,204 @@
-export default async function decorate(block) {
-  const buttons = document.createElement('div');
-  buttons.className = 'carousel-buttons';
-  [...block.children].forEach((row, i) => {
-    const classes = ['image', 'text'];
-    classes.forEach((e, j) => {
-      if (row.children[j]) row.children[j].classList.add(`carousel-${e}`);
-    });
-    /* buttons */
-    const button = document.createElement('button');
-    if (!i) button.classList.add('selected');
-    button.onclick = () => {
-      block.scrollTo({ top: 0, left: row.offsetLeft - row.parentNode.offsetLeft, behavior: 'smooth' });
-      [...buttons.children].forEach((r) => r.classList.remove('selected'));
-      button.classList.add('selected');
-    };
-    buttons.append(button);
-  });
+import { decorateIcons } from '../../scripts/lib-franklin.js';
+
+function template(props) {
+  return document.createRange().createContextualFragment(`
+    <div class="image-slideshow grid__container">
+      <div class="grid__structure">
+        <div class="grid__column grid__column--100 image-slideshow__wrapper">
+          <div>
+            <div class="thumbnail-slideshow thumbnail-slideshow--left-aligned">
+              
+              <!--
```

Comment:
Is this part commented out? If so, maybe we should remove it?

Reply:
I left it in because I'm not sure what it does yet. It's worth revisiting.

Reply:
I guess I can comment it in JS style so it's not rendered yes +1